### PR TITLE
Switch to using Conda

### PR DESCRIPTION
**Pull request by abejgonzalez**
This PR removes the different requirements files (for Python and platform-specific) in favor of using a Conda (what Chipyard/FireSim uses to manage multi-platform packages). In addition, it reworks the `README.md` to be clearer (docs on how to set up standalone FireMarshal, move links to the top, etc).

This PR builds off #236.

### CODE REVIEW

**Comment by abejgonzalez** [comment]
Posting this for initial thoughts / concerns

**Comment by abejgonzalez** [comment]
This was tested on a fresh Ubuntu VM on AWS so I'm confident this works!

**Comment by abejgonzalez** [comment]
Ping @timsnyder-siv @NathanTP

File: README.md [hunk 1]
```diff
@@ -1,88 +1,88 @@
...
-cat centos-requirements.txt | sudo xargs yum install -y
-```
+## Setup
 
-``ubuntu-requirements.txt`` is a list of packages for Ubuntu 18.04 that are needed by marshal.
-You can install these with:
-```
-cat ubuntu-requirements.txt | sudo xargs apt-get install -y
-```
+### ::RECOMMENDED:: Chipyard/FireSim Integration
```
Comment (jerryz123) [inline README.md:15]: I don't think this renders properly

File: README.md [hunk 2]
```diff
@@ -1,88 +1,88 @@
...
 
-Package names may be different on other distributions.
+The easiest way to use FireMarshal is to run it via [Chipyard](https://chipyard.readthedocs.io/en/latest/) or [FireSim](https://docs.fires.im/en/latest/).
+However, this is not required.
+To run FireMarshal independently, follow along from (#standalone-setup).
```
Comment (jerryz123) [inline README.md:19]: I don't think this creates a link in Github markdown. (IMO a link isn't necessary here
```suggestion
To run FireMarshal independently, follow along from the next section (Standalone setup).
```

**Review by jerryz123 — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (abejgonzalez) [reply README.md:15]: Since I can't bold the text I decided this makes sense to make the "recommended" word pop out more.

Comment (jerryz123) [reply README.md:15]: Try bold+italic: 

# *RECOMMENDED* stuff

**Review by jerryz123 — APPROVED** [review]

**Pull request merged**